build/dependencies.props

```diff
@@ -4,6 +4,7 @@
     <CoreFxVersion>4.3.0</CoreFxVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
```

Comment:
Good question. When I try to change this to the `BundledNETStandardPackageVersion`, I get the following warning probably because the `ci-dev` does not have this...this is because I had pinned the version earlier...

```
C:\Users\kichalla\.dotnet\x64\sdk\2.0.0-preview2-006082\NuGet.targets(98,5): warning : Dependency specified was NETStandard.Library.NETFramework (>= 2.0.0-preview2-25303-01) but ended up with NETStandard.Library.NETFramework 2.0.0-preview2-25304-02. [C:\github\testing\.build\KoreBuild.proj]
```